ChangeLog.d/getrandom.txt

```diff
@@ -0,0 +1,2 @@
+Features
+   Use glibc's getrandom when glibc > 2.15
```

Comment:
Suggested clarification: Use glibc's getrandom() instead of syscall() when glibc > 2.15.

(And a newline at the end to make our scripts happy.)